src/test/container-features/configs/lifecycle-hooks-advanced/.devcontainer/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM mcr.microsoft.com/devcontainers/base:ubuntu
```

Comment:
Curious, why not use an `image` property instead?

Reply:
There is no reason. I think I borrowed the skeleton of this test config from an existing one that had a dockerfile.